src/main/service/dataManipulationService.ts

```diff
@@ -311,4 +346,29 @@ export class DataManipulationService {
   public publicationDateInBst(publicationDatetime: string): string {
     return moment.utc(publicationDatetime).tz(this.timeZone).format('DD MMMM YYYY');
   }
+
+  /**
+   * Get the region name from the supplied location details
+   * Return either an empty string or the region name
+   * @param locationDetails The object to get the region name from
+   */
+  public getRegionNameFromLocationDetails(locationDetails: object): string {
```

Comment:
Just wondering if this method is worth it over doing the look up in the nunjucks file. Saves us from having extra methods / code